overlord/servicestate/quota_control_test.go

```diff
@@ -21,9 +21,11 @@ package servicestate_test
 
 import (
 	"fmt"
+	"os"
 	"path/filepath"
 	"time"
 
+	"gopkg.in/check.v1"
```

Comment:
check is already imported although with a `.` so its exported types/functions/variables become directly visible without the package prefix

Reply:
This is still a problem, I suggest to drop the new import and use the symbols that become available through `. "gopkg.in/check.v1"` just below.

```suggestion
```